pkg/controller/config/config_controller.go

```diff
@@ -218,12 +223,15 @@ func (r *ReconcileConfig) Reconcile(request reconcile.Request) (reconcile.Result
 	r.watched.Replace(newSyncOnly)
 
 	// *Note the following steps are not transactional with respect to admission control*
-
```

Comment:
nit: we should preserve the newline here to highlight the distinction between a high-level comment about the overall code that follows and a specific comment about the immediately following line.